### PR TITLE
Registration page/styles/mobile

### PR DESCRIPTION
**Pull request by Klypalskyi**
на компоненте GoogleButton стоит закомментированный редирект. 
Как заглушка работает console.log
image

### CODE REVIEW

File: src/Containers/App/App.jsx [hunk 1]
```diff
@@ -1,15 +1,17 @@
-import React, { useState, useEffect } from 'react';
-import CssBaseline from '@material-ui/core/CssBaseline';
+import React from 'react'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import Auth from '../../pages/Auth/Auth'
```
Comment (vbguard) [inline src/Containers/App/App.jsx:3]: remove from this component your component

File: src/Containers/App/App.jsx [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 
 // import PropTypes from 'prop-types'
 
-function App() {
+function App () {
   return (
     <>
       <CssBaseline />
+      <Auth />
```
Comment (vbguard) [inline src/Containers/App/App.jsx:11]: remove

File: src/components/GoogleButton/GoogleButton.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+// import url from '../../api/apiEndpoint';
+
+const GoogleButton = () => {
+  const handleClick = () => {
+    // document.location.replace(url.googleAuth);
+    console.log('go to auth by google')
+  }
+
+  return (
+    <button type='button' className={styles.button} onClick={handleClick}>
```
Comment (vbguard) [inline src/components/GoogleButton/GoogleButton.jsx]: it's tag a not button. `<a href="https://domain.com/api/v1/auth/google">Google</a>`

File: src/index.js [hunk 4]
```diff
@@ -3,23 +3,63 @@ import React from 'react';
...
 import * as serviceWorker from './serviceWorker';
 import './assets/css/normalize.css';
 import './assets/css/fonts.css';
 import './assets/css/main.css';
+import './assets/css/var.css';
 
 // const initStore = {};
 const store = createStore;
 
+const theme = createMuiTheme({
```
Comment (vbguard) [inline src/index.js]: we have theme.json

File: src/pages/Auth/Auth.jsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Switch, Route, Redirect } from 'react-router-dom'
+import RegistrationPage from '../../components/RegistrationPage/RegistrationPage'
+
+function Auth () {
+  return (
+    <Switch>
+      <Route exact path='/'>
+        <Redirect to='/registration' />
```
Comment (vbguard) [inline src/pages/Auth/Auth.jsx]: redirect to login

**Review by vbguard — COMMENTED** [review]
need improve, pull new dev

File: src/index.js (hunk 4 above)
Comment (Klypalskyi) [reply src/index.js]: that theme.json is not working for now. And that part will be resolved on themeProveider branch soon :)

File: src/pages/Auth/Auth.jsx (hunk 5 above)
Comment (Klypalskyi) [reply src/pages/Auth/Auth.jsx]: Done

File: src/components/GoogleButton/GoogleButton.jsx (hunk 3 above)
Comment (Klypalskyi) [reply src/components/GoogleButton/GoogleButton.jsx]: Done

File: src/Containers/App/App.jsx (hunk 2 above)
Comment (Klypalskyi) [reply src/Containers/App/App.jsx:11]: Done

File: src/Containers/App/App.jsx (hunk 1 above)
Comment (Klypalskyi) [reply src/Containers/App/App.jsx:3]: Done

**Review by vbguard — APPROVED** [review]

**Pull request merged**